canonical/ubuntu.com
# Update chart data on /kernel/lifecycle

## Done

- Updates chart data on /kernel/lifecycle based on [spreadsheet data](https://docs.google.com/spreadsheets/d/1KIuOAa620yseE0uhIl67u2SlS0bWlPI_d98bvYM2Yl8/edit#gid=281992953) & some other data for other charts that were broken
- Updates `createChart` to use the earliest and latest dates to scale the graph domain
- Updates tabbed-content.js to use the `u-hide` class instead of `hidden`
- Updates pages that used `tabbed-content.js` to work with refactor
- updates `_pattern_tabs.scss` to have a 'on-paper' vertical tab pattern
- Fixes tabs on `openstack/install`


## QA

- Each graph is build of a different tab in the [spreadsheet](https://docs.google.com/spreadsheets/d/1KIuOAa620yseE0uhIl67u2SlS0bWlPI_d98bvYM2Yl8/edit#gid=281992953), compare the data with each graph tab in [the demo](https://ubuntu-com-12858.demos.haus/kernel/lifecycle) as follows
[Kernel all](https://docs.google.com/spreadsheets/d/1KIuOAa620yseE0uhIl67u2SlS0bWlPI_d98bvYM2Yl8/edit#gid=703230516) -> `All releases`
[Kernel lts](https://docs.google.com/spreadsheets/d/1KIuOAa620yseE0uhIl67u2SlS0bWlPI_d98bvYM2Yl8/edit#gid=281992953) -> `All LTS Releases`
[Kernel 22.04](https://docs.google.com/spreadsheets/d/1KIuOAa620yseE0uhIl67u2SlS0bWlPI_d98bvYM2Yl8/edit#gid=1844838308) -> `22.04`

- check out the other pages I made changes to from the description and files changed and check everything works as before

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-2248
